packages/foundry-sdk-generator/src/ontologyMetadata/ontologyMetadataResolver.ts

```diff
@@ -51,6 +52,12 @@ export class OntologyMetadataResolver {
       ) => expectedEntities.objectTypes.has(objectTypeApiName.toLowerCase())),
     );
 
+    const filteredInterfaceTypes = Object.fromEntries(
```

Comment:
Can we only add these for 2.x? That way you have to get all the way to 2.0 to use them and we can reduce 1.x support burden.